File: 2785-sort-vowels-in-a-string/2785-sort-vowels-in-a-string.cpp
class Solution {
public:
    string sortVowels(string s) {
        string a = "";
        int j = 0;
        string vow = "AEIOUaeiou";
        for(int i = 0 ; i < s.size(); ++i){
            for(const char k: vow){
                if(s[i] == k){
                    s[i] = '#';
                    a+= k;
                }
            }
        }
        sort(a.begin(), a.end());
        for(int i = 0 ; i < s.size(); ++i){
            if(s[i] == '#'){
                s[i] = a[j];
                ++j;
            }
        }
        return s;
    }
};

File: 2785-sort-vowels-in-a-string/2785-sort-vowels-in-a-string.py
class Solution:
    def sortVowels(self, s: str) -> str:
        v = "AEIOUaeiou"
        a = []
        b = ""
        for i in s:
            if i in v:
                a.append(i)
        a.sort()
        j = 0
        for i in s:
            if i in v:
                b += a[j]
                j += 1
            else:
                b += i
        return b
        